index.html

```diff
@@ -13,10 +13,10 @@
     <body>
         <header>
             <div class="logo">
-                <img id="logo_picture" src="C:\Users\12083\Documents\GitHub\Microverse-My-Portfolio\img\Logo Placeholder.png" alt="My_Logo">
+                <img id="logo_picture" src="img/LogoPlaceholder.png" alt="My_Logo">
             </div>
             <nav class="menu-icon">
-                <img id="hamburger_menu" src="C:\Users\12083\Documents\GitHub\Microverse-My-Portfolio\img\Union.png" alt="menu_button">
+                <img id="hamburger_menu" src="img/Union.png" alt="menu_button">
             </nav>
         </header>
```

Comment:
- [ ] Good job, you are using header and nav, but please make your logo clickable, add a link `a` tag to your logo